petab/parameter_mapping.py

```diff
@@ -336,8 +336,15 @@ def _apply_condition_parameters(par_mapping: ParMappingDict,
         if overridee_id == CONDITION_NAME:
             continue
 
+        # Species and compartments are handled elsewhere
+        if sbml_model.getSpecies(overridee_id) is not None:
+            continue
+        if sbml_model.getCompartment(overridee_id) is not None:
+            continue
+
         par_mapping[overridee_id] = core.to_float_if_float(
             condition_df.loc[condition_id, overridee_id])
+        scale_mapping[overridee_id] = LIN
```

Comment:
yes, should work. we assume that all values are in lin space, and for condition specific numeric overrides, we will never need scales, as these are not estimated.

Reply:
Yes, that was so far incoherent. Scale mappings had potentially fewer entries than parameter value mapping.